### DISCUSSION TITLE
When will the Self Hosting version support multiple workspaces?

### DISCUSSION THREAD

**Discussion by ClearSeve** [outline/outline#6806] [Self Hosting]
In my opinion, multiple workspaces are a relatively basic function for individuals or enterprises. I can use workspaces to separate work, life, and personal hobbies.

**Comment by phil-flip** [comment]
Same here. I really love Outline, but it is one of the things holding me back from using it for my little side-gig.
Collections are nice, but the sidebar can get crowded quite easily.
According to this discussion https://github.com/outline/outline/discussions/4531 they are only planning to release it for EE and i totally get their reasoning, as it is something that can be justified as an enterprise thing.
Asking again to be sure might not hurt. As I don't see this feature in the pricing-breakdown.

**Comment by sandros94** [comment]
Yeah I mean, neither I see a reason for not being in the self hosted version.
Since anyone could simply redeploy the same instance multiple times for separating work, hobby and personal (what I'm currently doing).

**Reply by Ultragian** [reply]
That's just bad. Imagine setting up 3 Nextcloud instances lmao

**Reply by sandros94** [reply]
@Ultragian yeah I mean, if you just deploy a docker stack you could deploy it as many time as your resources will allow for...

**Comment by REDCOREBE** [comment]
Hello,

Just following up on this topic as it’s a real blocker for us.

I really love Outline’s approach (UX, simplicity, documentation workflow), but the lack of multiple workspaces in the self-hosted version prevents wider adoption on our side. In professional environments, information segmentation (clients, projects, teams, confidentiality levels) is essential.

While the cloud version supports this, it’s not an option for us due to data handling and confidentiality requirements.

Even a limited or progressive implementation of workspaces in the self-hosted version would make Outline much more viable for professional use cases.

Thanks for the great work and for listening to the community.

**Reply by tommoor** [reply]
Workspaces in cloud have completely separate users and settings, is this your desire for self hosted too?

I feel like I'd actually a different concept that's needed

**Reply by REDCOREBE** [reply]
On my side, I’d mainly like to segment information between clients / departments at the workspace level.

Regarding users, I’d prefer a hybrid model:
- cross-workspace users, who can access multiple workspaces,
- and users defined exclusively for one or several specific workspaces (depending on the need).

Having one workspace per client or department would be much more user-friendly and intuitive.

Right now, we’re working within a single workspace, but it’s not very natural—especially when considering the number of clients / departments we might have in the future. Also, since there won’t be a search on collection names, but only on the content across all collections, it becomes difficult to navigate if everything is grouped into one workspace.
